src/ray-controller.js

```diff
@@ -79,7 +80,15 @@ export default class RayController extends EventEmitter {
 
     var gamepad = this.getVRGamepad_();
 
+    this.gamepad = gamepad;
     if (gamepad) {
+      if(gamepad.hand){
+        if(gamepad.hand === 'left'){
```

Comment:
I'd replace these couple lines with this:

```js
this.isLeftHanded = gamepad.hand === 'left';
```

Reply:
With parens is even better:

    this.isLeftHanded = (gamepad.hand === 'left');